repl_mon.c

```diff
@@ -136,6 +136,33 @@ repl_mon_init()
             elog(FATAL, "Error while creating table");
     }
 
+    /* Make repl_mon settings visible on replicas */
+    initStringInfo(&buf);
+    appendStringInfo(&buf, "CREATE TABLE IF NOT EXISTS public.repl_mon_settings ("
```

Comment:
Let's concatenate `tablename` and `_settings` as it's done abode instead of hardcoding table name.